src/Components/MainStatsDisplay/MainStatsDisplay.js

```diff
@@ -29,25 +27,61 @@ function MainStatsDisplay({ selectedParts }) {
+
       { name: "Total Weight", value: totalStats.total_weight },
+
       { name: "Total Arms Load", value: totalStats.total_arms_load },
       { name: "Arms Load Limit", value: totalStats.arms_load_limit },
       { name: "Total Load", value: totalStats.total_load },
       { name: "Load Limit", value: totalStats.load_limit },
       { name: "Total EN Load", value: totalStats.EN_load },
       { name: "EN Output", value: totalStats.EN_output },
+
       { name: "Current Load", value: totalStats.total_load, isBar: true, maxValue: totalStats.load_limit },
       { name: "Current Arms Load", value: (totalStats.total_arms_load), isBar: true, maxValue: totalStats.arms_load_limit },
       { name: "Current EN Load", value: totalStats.EN_load, isBar: true, maxValue: totalStats.EN_output },
+      /* Temporary display of advanced stats */
+
+      /* Amount of raw damage that can be sustained */
+      { name: "Effective AP (vs kinetic)", value: totalStats.effective_AP_kinetic },
+      { name: "Effective AP (vs energy)", value: totalStats.effective_AP_energy },
+      { name: "Effective AP (vs explosive)", value: totalStats.effective_AP_explosive },
+      { name: "Effective AP (average)", value: totalStats.effective_AP_avg },
+      /* Time it takes for the generator to recover an energy amount equal to one quick boost.
+         The AC can quick boost indefinitely if the interval between boosts is at least equal
+         to this value */
+      { name: "Infinite QB interval", value: totalStats.infinite_qb_interval, roundTo: 0.01 },
+      /* Time for EN recovery to start when the energy is fully depleted */
+      { name: "EN Recharge Delay (redline)", value: totalStats.EN_recharge_delay_redline, roundTo: 0.01 },
+      /* Time to fully recharge energy when not redlining. This value is theoretical because
+         obviously if energy needs to be recovered fully it means the generator is redlining */
+      { name: "Full recharge time", value: totalStats.full_recharge_time, roundTo: 0.01 },
+      /* Time to fully recover energy when redlining */
+      { name: "Full recharge time (redline)", value: totalStats.full_recharge_time_redline, roundTo: 0.01 },
+      /* % of weight allocated to units (weapons) */
+      { name: "Units weight %", value: totalStats.group_weight_perc[0]},
+      /* % of weight allocated to body parts */
+      { name: "Body weight %", value: totalStats.group_weight_perc[1]},
+      /* % of weight allocated to inner parts */
+      { name: "Inner parts weight %", value: totalStats.group_weight_perc[2]},
+      /* % of EN load allocated to units (weapons) */
+      { name: "Units EN load %", value: totalStats.group_EN_load_perc[0]},
+      /* % of EN load allocated to body parts */
+      { name: "Body EN load %", value: totalStats.group_EN_load_perc[1]},
+      /* % of EN load allocated to inner parts */
+      { name: "Inner parts EN load %", value: totalStats.group_EN_load_perc[2]}
```

Comment:
The weight / EN load % values could be shown in bars similar to how GitHub shows the breakdown of languages in a repo
![image](https://github.com/user-attachments/assets/b3fe13b1-a9e2-4183-9f03-2e15a3669699)